tesla_http_proxy/DOCS.md

```diff
@@ -54,10 +119,23 @@ nxjvrm2M6uKfDEYS52ITVVbzqGMzzbKCO/tuu78432jU6Z96BNR8NSoRXg==
 -----END PUBLIC KEY-----
 ```
 
-You should have a config file at `/share/nginx_proxy/nginx_tesla.conf` that hosts the static public key file.  You may need to modify this file depending on your SSL config.  
+- You should have a config file at `/share/nginx_proxy/nginx_tesla.conf` that hosts the static public key file. You may need to modify this file depending on your SSL config.
+
+- If you get `login_required` error when trying to send API commands, it's likely because you tried to reuse the refresh token more than once. Try fully removing the Tesla integration from HA and adding it back again.
+
+- When you enroll the public key in the vehicle, if you don't get a prompt, try moving within BLE range of the vehicle.
```

Comment:
I didn't think BLE connection to the car was a requirement but I could be wrong... perhaps this was mentioned somewhere in a Discussion?!

Reply:
This comment is done by the author of the repo, not me.
It probably needs to be done in order to wake the car

Reply:
I don't know why BLE is relevant, but it was mentioned by an Android user here: https://github.com/llamafilm/tesla-http-proxy-addon/discussions/58#discussioncomment-8889866